Tools/MonoGame.Effect.Compiler/MonoGame.Effect.Compiler.csproj

```diff
@@ -3,13 +3,15 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Description>This package provides you with MonoGame Framework Effect Compiler (MGFX) tool used for packaging content in to xnb files.</Description>
```

Comment:
```suggestion
    <Description>The MonoGame Framework Effect Compiler (MGFX) tool used to compile shaders for use with MonoGame.</Description>
```

Reply:
`The MonoGame Framework Effect Compiler (MGFXC) command line tool used to compile shaders.`